DEBUG 2017-01-31 09:30:08,951: 	 Executing command: 'submit'
DEBUG 2017-01-31 09:30:08,971: 	 Found CRAB cache file /home/georgemm01/.crab3
DEBUG 2017-01-31 09:30:09,004: 	 Checking credentials
DEBUG 2017-01-31 09:30:09,010: 	 Found proxy file /home/georgemm01/.x509up_u1452
DEBUG 2017-01-31 09:30:09,010: 	 Getting proxy life time left
DEBUG 2017-01-31 09:30:09,115: 	 Executing : 
 command : voms-proxy-info -file /home/georgemm01/.x509up_u1452 -timeleft
 output : 606606

 error: 
 retcode : 0
DEBUG 2017-01-31 09:30:09,221: 	 Executing : 
 command : voms-proxy-info -file /home/georgemm01/.x509up_u1452 -actimeleft
 output : 606605

 error: 
 retcode : 0
DEBUG 2017-01-31 09:30:09,222: 	 Proxy valid for 168:30:06 hours
INFO 2017-01-31 09:30:09,222: 	 Will use CRAB configuration file crabConfig_tutorial_MC_analysis_def.py
DEBUG 2017-01-31 09:30:09,223: 	 Loading CRAB configuration file.
DEBUG 2017-01-31 09:30:11,537: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
DEBUG 2017-01-31 09:30:11,677: 	 Executing : 
 command : openssl x509 -in ~/.globus/usercert.pem -subject -noout
 output : subject= /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=moralesm/CN=725145/CN=Jorge Daniel Morales Mendoza

 error: 
 retcode : 0
DEBUG 2017-01-31 09:30:11,677: 	 Checking credentials
DEBUG 2017-01-31 09:30:11,678: 	 Found proxy file /home/georgemm01/.x509up_u1452
DEBUG 2017-01-31 09:30:11,678: 	 Getting proxy life time left
DEBUG 2017-01-31 09:30:11,783: 	 Executing : 
 command : voms-proxy-info -file /home/georgemm01/.x509up_u1452 -timeleft
 output : 606604

 error: 
 retcode : 0
DEBUG 2017-01-31 09:30:11,888: 	 Executing : 
 command : voms-proxy-info -file /home/georgemm01/.x509up_u1452 -actimeleft
 output : 606603

 error: 
 retcode : 0
DEBUG 2017-01-31 09:30:11,889: 	 Proxy valid for 168:30:04 hours
DEBUG 2017-01-31 09:30:11,996: 	 Executing : 
 command : env X509_USER_PROXY=/home/georgemm01/.x509up_u1452 voms-proxy-info -fqan
 output : /cms/Role=NULL/Capability=NULL
/cms/uscms/Role=NULL/Capability=NULL

 error: 
 retcode : 0
INFO 2017-01-31 09:30:13,293: 	 The server answered with an error.
INFO 2017-01-31 09:30:13,294: 	 It seems the CMSWEB frontend is not responding. Please check: https://twiki.cern.ch/twiki/bin/viewauth/CMS/ScheduledInterventions
DEBUG 2017-01-31 09:30:13,294: 	 ERROR: Service Unavailable (503):  '<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
 <head>
 <title>CMSWEB Error: Service unavailable</title>
 <meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
 <link rel="shortcut icon" href="/favicon.ico" type="image/x-icon" />
 <link rel="stylesheet" type="text/css" media="screen" href="/css/cmsweb.css" />
</head>
<body>
 <div id="main">
  <div id="top">
   <div class="boxTitle"><img src="/img/title.gif" alt="CMSWEB (title)" /></div>
  </div>
  <div id="middle">
   <div class="boxLinkContainer">
    <h2>Service unavailable</h2>
    <p>The site you requested is not available.</p><p>CMS <a href='https://twiki.cern.ch/twiki/bin/view/CMS/ScheduledInterventions'>scheduled interventions page</a> lists planned CMS and CERN service downtimes and known site-wide computing system incidents. Planned updates are notified on the <a href='https://hypernews.cern.ch/HyperNews/CMS/get/cernCompAnnounce.html'>CERN computing announcements</a> forum.</p><p>Would you please be so kind and <a href='https://ggus.eu/?mode=ticket_cms'>create a ticket</a> to"CMS Web Tools" support unit</a> if this interruption was unexpected?</p>
   </div>
  </div>
  <div id="bottom">
   <div class="boxFooter">&nbsp;</div>
  </div>
 </div>
</body>
</html>
'
DEBUG 2017-01-31 09:30:13,294: 	 Command failed with URI: https://cmsweb.cern.ch/crabserver/prod/info
DEBUG 2017-01-31 09:30:13,294: 	      Input data: {'subresource': 'delegatedn'}
DEBUG 2017-01-31 09:30:13,294: 	      Request headers: {'Date': 'Tue, 31 Jan 2017 15:30:13 GMT', 'Connection': 'close', 'Content-Type': 'text/html', 'CMS-Server-Time': 'D=11694 t=1485876613206916', 'Server': 'Apache'}
ERROR 2017-01-31 09:30:13,294: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1611-comp3/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1611-comp3/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1611-comp3/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1611-comp3/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 339, in __init__
    self.handleProxy(proxyOptsSetPlace)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1611-comp3/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 414, in handleProxy
    all_task_workers_dns = server_info('delegatedn', self.serverurl, self.proxyfilename, baseurl)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1611-comp3/lib/python2.7/site-packages/CRABClient/ClientUtilities.py", line 664, in server_info
    dictresult, dummyStatus, dummyReason = server.get(baseurl, requestdict)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1611-comp3/lib/python2.7/site-packages/RESTInteractions.py", line 79, in get
    return self.makeRequest(uri = uri, data = data, verb = 'GET')
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1611-comp3/lib/python2.7/site-packages/RESTInteractions.py", line 128, in makeRequest
    capath=caCertPath)#, verbose=True)# for debug
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1611-comp3/lib/python2.7/site-packages/WMCore/Services/pycurl_manager.py", line 207, in request
    raise exc
HTTPException: url=https://cmsweb.cern.ch/crabserver/prod/info, code=503, reason=Service Unavailable, headers={'Date': 'Tue, 31 Jan 2017 15:30:13 GMT', 'Connection': 'close', 'Content-Type': 'text/html', 'CMS-Server-Time': 'D=11694 t=1485876613206916', 'Server': 'Apache'}
